modules/doi/modules/doi_importer/includes/importer.inc

```diff
@@ -24,38 +24,44 @@ class DOIImporter extends IslandoraBatchImporter {
    *
    * @see IslandoraBatchImporter::getForm()
    */
-  public static function getForm(array &$form_state) {
-    return array(
-      'fs' => array(
-        '#type' => 'fieldset',
-        '#title' => t('DOI Batch Journal Importer'),
-        '#description' => t('Provide either a list of Digital Object Identifiers in a file, or enter DOIs in the textarea below. In either case, the DOIs in the list should be separated by either whitespace or commas (or some combination thereof). NOTE: Presently, only journal (articles) are supported.'),
-        'file' => array(
-          '#type' => 'managed_file',
-          '#title' => t('File of DOIs'),
-          '#description' => t('A file listing Digital Object Identifiers (with ".txt" extension).'),
-          '#upload_validators' => array(
-            'file_validate_extensions' => array('txt'),
+  public static function getForm(array & $form_state) {
+    $openurl_pid = variable_get('islandora_doi_openurl_pid', 'user@example.com');
```

Comment:
Just to be sure  "OpenURL PID email address" == "crossref registered email address". We do have somewhere that explains this or is this a thing Scholar module users are 100% aware?